lua/telescope/pickers/layout_strategies.lua

```diff
@@ -261,9 +274,15 @@ layout_strategies.vertical = function(self, max_columns, max_lines)
   results.col, preview.col, prompt.col = width_padding, width_padding, width_padding
 
   if self.previewer then
-    preview.line = height_padding
-    results.line = preview.line + preview.height + 2
-    prompt.line = results.line + results.height + 2
+    if not config.values.layout_defaults.vertical.mirror then
```

Comment:
Shouldn't this use `layout_config` not `config.values`?

Reply:
When I tried `layout_config` instead of `config.values`, it was returning a `nil` value to me. I can double check that I didn't make a mistake there, but it wasn't working until I switched to using `config.values`.